common/component_interface_tools/package.xml

```diff
@@ -15,9 +15,11 @@
   <depend>diagnostic_updater</depend>
   <depend>fmt</depend>
   <depend>rclcpp</depend>
+  <depend>service_log_checker</depend>
```

Comment:
The `service_log_checker` is not a package. This causes CI to fail.

Reply:
This was removed.